test/t8n/t8n.cpp

```diff
@@ -0,0 +1,177 @@
+#include "../state/mpt_hash.hpp"
+#include "../state/rlp.hpp"
+#include "../statetest/statetest.hpp"
+#include <evmone/evmone.h>
+#include <nlohmann/json.hpp>
+#include <filesystem>
+#include <fstream>
+#include <iostream>
+#include <string_view>
+
+namespace fs = std::filesystem;
+namespace json = nlohmann;
```

Comment:
Why not?
```suggestion
namespace json = nlohmann::json;
```

Would remove the `json::json::...` cases.

Reply:
Ah ok, this is consistent with `statetest_loader`, can change later then.

Reply:
Right, can we just include the namespace (`using namespace nlohmann`)?